lib/main.js

```diff
@@ -105,14 +115,26 @@ ava.test.before(async () => {
   await utils.waitUntil(() => {
     return resinio.isDeviceOnline(placeholder.uuid)
   })
+  context.uuid = placeholder.uuid
 
   console.log('Waiting while supervisor starts')
   await utils.waitUntil(async () => {
     return await resinio.getDeviceStatus(placeholder.uuid) === 'Idle'
   })
 
+  const uptime = await utils.getDeviceUptime((command) => {
+    return resinio.sshHostOS(command, context.uuid, context.key.privateKeyPath)
+  })
+
+  console.log('Gather metrics')
+  results.imageSize = `${fs.statSync(imagePath).size / 1000000.0} Mb`
+  results.provisionTime = `${Math.floor(uptime / 60)}m ${uptime % 60}s`
+
   console.log('Running tests:')
-  context.uuid = placeholder.uuid
+})
+
+ava.test.after.always(async () => {
+  fse.outputJsonSync(options.resultsFilePath, results)
```

Comment:
Should this be `fs`?

Reply:
We already have fse imported, why not use the helper function?

Reply:
Ah, I didn't see it. Never-mind then :)